src/graph/entities/node.c

```diff
@@ -46,3 +46,10 @@ void Node_Free(Node* node) {
 	free(node);
 	node = NULL;
 }
+
+void Node_Print(const Node *node, FILE *out) {
+    if(!out) out = stdout;
+    GraphEntity_Print((GraphEntity*) node, GETYPE_NODE, 0);
+    if (node->label || node->alias)
```

Comment:
The intention here was to print if either values is available. A null value is printed as "(null)". Although this is diagnostic code, we can add code to print empty strings instead of nulls.